KarhooSDK/Service/Quote/Interactor/CoverageInteractor.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  CoverageInteractor.swift
+//  KarhooSDK
+//
+//  Created by Nurseda Balcioglu on 10/11/2020.
+//  Copyright © 2020 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+
+protocol CoverageInteractor: KarhooExecutable {
```

Comment:
Now that I think about it, maybe rename all instances of Coverage to QuoteCoverage instead. Coverage is very vague on its own so may want to be specific with the name and call it QuoteCoverage.

Reply:
@jeremyauclert if we're planning on keeping naming in line on both platforms then we probably need to do this for Android as well.